### ISSUE TITLE
Character verification failed

### ISSUE THREAD

**Issue by D0N2k** [exodus4d/pathfinder#760]
Hey,

i just upgraded to 1.5 with your Upgrade Guide.
Now with 1.5 nobody gets tracked the pilot counter is 0 all the time.
And everytime i go back to Home it says
"Character verification failed: This can happen through "invalid cookies(SSO)", "login restrictions", "ESI problems"."

**Comment by tralafiti** [comment]
Try reinstalling the composer dependencies:
* remove `composer.lock`
* remove `vendor `folder
* `composer install`

Since the lock-file (sadly) isn't versioned in git, it's possible that you don't get the newest dependencies (here pathfinder_esi) if you update a preexisting instance. Possible that a `compser update` is enough too if you don't want to delete files.

**Comment by D0N2k** [comment]
I did composer update.
Composer suggested it after running composer install.

I did it while installing. But ill try reinstalling the dependencies.

**Comment by tralafiti** [comment]
If it doesn't help you can still peek into the php error logs :)

**Comment by dagoaty** [comment]
I have the same issue, but only after following upgrade step 7.

> 7. Cleanup DB
> 
> Some tables and columns are no longer needed and should be deleted manually from pathfinder DB
> Drop column character.esiAccessToken
> Drop column character.esiAccessTokenExpires
> Drop column character.esiRefreshToken

Prior to deleting these columns everything was working fine. Also, in `/setup` these columns get readded so I think these weren't meant to be deleted.

I've no idea how to fix this situation, but hopefully @exodus4d might be able to help with instructions how to repopulate these columns.

**Comment by dagoaty** [comment]
I've managed to get my install working again. Maybe these steps will work for you too, @D0N2k 

1. Go back into the `/setup` page and rerun **"Setup Tables** against the `pathfinder` DB. This will recreate the missing three columns.
2. If you are using the optional Redis cache, click **Flush** to wipe the stored keys

Once I had done that the access tokens for my characters were able to be repopulated when I logged in.

**Comment by exodus4d** [comment]
@dagoaty You are right. die "Drop column" instructions are wrong. I fixed them some days ago. 
I renamed 3 columns:

| old | new  | 
| ------------- |-------------|
| `character.crestAccessToken`  |  `character.esiAccessToken` |
| `character.crestAccessTokenUpdated`  |  `character.esiAccessTokenExpires` |
| `character.crestRefreshToken`  |  `character.esiRefreshToken` |

My intention was to create the 3 new columns on `/setup` **"Setup Tables"**, and drop the `crest*` columns manually. 
If you dropped the new `esi*` columns manually right after the creation, that is not a big problem. Re-run  the **"Setup Tables"** button will re-create them... my bad :(

To make it more obvious, which columns exist in your DB that are no longer required (deprecated), the DB _diff_ panel on `/setup` now lists them as `deprecated` if they are no longer part of the _Model_ Class:

![LUu7trz](https://user-images.githubusercontent.com/9299033/56454960-da2a7c00-6358-11e9-9603-42c5c5e2a00b.png)
